src/main/webapp/WEB-INF/views/UserPage.html

```diff
@@ -0,0 +1,376 @@
+                                        <a href="#"><h4 class="card-title">The NOMATIC Backpack and Travel Pack</h4></a>
+                                        <div class="card-text">
+                                            <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                                                tempor
+                                                incididunt ut labore et
+                                                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                                                ullamco
+                                                laboris nisi ut aliquip
+                                                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                                                voluptate
+                                                velit
+                                                esse cillum dolore eu
+                                                fugiat nulla pariatur.
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="card-footer">
+                                        <div class="row">
+                                            <div class="col-xs-6">
+                                                <div>min</div>
+                                                <div><span class="fa fa-dollar"></span>100000</div>
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><span class="fa fa-dollar"></span>9999</div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="col-xs-12">
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-9 cta-contents">
+                    <a href="#"><h1 class="cta-title">Projects</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#">My Projects</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addProject">
+                        <fieldset>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add a <legend> tag to this fieldset. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)